dev-packages/application-manager/src/generator/webpack-generator.ts

```diff
@@ -38,7 +39,8 @@ const { mode }  = yargs.option('mode', {
 }).argv;
 const development = mode === 'development';${this.ifMonaco(() => `
 
-const monacoEditorPath = development ? '${this.resolve('monaco-editor-core', 'dev/vs')}' : '${this.resolve('monaco-editor-core', 'min/vs')}';
+const monacoEditorPath = development ? '${this.resolve('monaco-editor', 'dev/vs')}' : '${this.resolve('monaco-editor', 'esm/vs')}';
```

Comment:
Why did you add monaco-editor again? Is it needed?

Reply:
it is a left over from the various tests before, nice catch !